docs/userGuide/contentAuthoring.md

```diff
@@ -173,6 +173,56 @@ or
 ...
 </div>
 ```
+
+#### Question body syntax
+
+<panel header=":lock::key: Does MarkBind allow using checkboxes and radio buttons with the Question and Panel components?">
+<question>
+
+- ( ) Yes
+- ( ) No
+
+<div slot="hint">
+Probably yes
+</div>
+<div slot="answer">
+Yes!
+<br><br>
+
+```html
+<panel header=":lock::key: Does MarkBind allow for the use of checkboxes and radio buttons?">

```

Comment:
Is it a good idea to put the sample source code inside the answer slot? They might miss it if they do not expand the answer slot.